framework/Objects/Rest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace JustCoded\WP\Framework\Objects;
+
+/**
+ * Custom rest class to simplify the process of registering rest endpoints.
+ * Options are rewritten to be more simple for understanding.
+ * However they affect all standard options from here:
+ * https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/
+ */
+abstract class Rest {
```

Comment:
Let's create it like a RestController.
So it can register multiple routes.
Namespace should be set as usual property.
We don't need constants, because WP has there own.

You need to have abstract routes() method, where you can add routes like:
$this->add_route('GET', 'asdf', [$this, 'view'], $args);

Also we need a static method like get_permalink($methodName, $args), which will generate full permalink with namespace and route based on regexp pattern and args passed